src/app/feedback/feedback-widget/feedback-widget.component.ts

```diff
@@ -26,9 +26,20 @@ export class FeedbackWidgetComponent implements OnInit {
   @HostListener('window:mousedown')
   handleDialogClose() {
     // TODO: Move out to a directive
-    const belongsToPopup = event['path'].some(item =>
-      item.className && item.className.includes('feedback-container')
-    );
+    function findMatchingDOMAncestor(element) {
+      while (element.parentNode) {
+        if (element.className && element.className.indexOf('feedback-container') >= 0) {
+          return true;
+        }
+        element = element.parentNode;
```

Comment:
I'm trying to understand what happens if there is no element with feedback-container class in the end. Will it throw null doesn't have a property parentNode error or something?